### ISSUE TITLE
NTR: complement component C1q complex

### ISSUE THREAD

**Issue by bmeldal** [geneontology/go-ontology#18150]
I'm requesting a sub-complex of complement component C1 complex that has additional roles on it's own.

Name:
complement component C1q complex

Definition:
A protein complex composed of six subunits of each of the three homologous polypeptide chains C1QA, C1QB, and C1QB. 

Subunit of complement component C1 where it recognises immune complexes, or other molecules, and triggers the classical pathway of the complement system. Independent of complement activation, C1q appears to have additional roles in homeostasis and cellular development, superoxide (O2-) production by neutrophils, blood coagulation and neurological synapse pruning. 

Synonyms:
Complement 1q [EXACT]
C1q [EXACT]

Relationships:
GO:0005602 has_part NEW term
is_a GO:0032991 protein-containing complex
part_of GO:0005615 extracellular space

References:
PMID:29449492
GOID:bhm

CAUTION:
I am making all complement complexes over the next few weeks/months (?!?!?). Should we rather make a group term for all compliments? The only issue is that most, but not all, are serine proteases. E.g. C1q is NOT an enzyme but C1 is...

Birgit

Note to self: EBI-6264901/CPX-1919

**Comment by ukemi** [comment]
@addiehl, do you have an opinion on this? Would it make sense to group them all? If so, what would be the grouping element?

**Comment by ukemi** [comment]
@bmeldal, I'm adding the term, but how should it relate to the binding and receptor terms?

**Comment by bmeldal** [comment]
Heads up:
I'm also working on the factors of the coagulation system (Factor V - XIII and associated proteins). They are all dimers of 2 processed chains and some form further complexes, e.g. FV+FX.

It will be a new ticket. If Alex wants to comment on the Factors I will tag him in my ticket when I make it.

**Comment by ukemi** [comment]
#18151

**Comment by bmeldal** [comment]
> I'm adding the term, but how should it relate to the binding and receptor terms?

Sorry, there are no binding or receptor terms in my request. What am I missing?

**Comment by ukemi** [comment]
They are already in the ontology. If they refer to the complex, they should relate to it in some way.

**Comment by bmeldal** [comment]
The complex doesn't have receptor or binding activity with itself. A GP can either be part of the C1q complex or have the C1q receptor or binding activity if it interacts with C1q. I don't know how to link them in the ontology.

Can you give a related example where you have such a link in the ontology?

**Comment by addiehl** [comment]
I'm not sure a grouping term for all complement component complexes makes sense, except on the lexical level.  Complement proteins fall into several protein families and form a number of structurally and functionally distinct complexes.  One could imagine grouping terms for the C3-convertase complexes or the C5 convertase complexes of the classical and alternative pathways, which themselves might be further grouped.  But the C1-complex is distinct structure and function, as is the membrane attack complex.

At least three complement related complexes already exist in the GO, and I certainly favor adding additional complexes.  If I remember correctly, my fix to the membrane attack complex definition in 2003 was my very first GO issue tracker item, so it is great to see this effort, since most of my later work in this area was on complement pathway related processes rather than complexes.

**Comment by ukemi** [comment]
Thanks @addiehl

**Comment by ukemi** [comment]
'phosphoprotein binding' in GO has a formal definition of: binding and 'has input' some phosphoprotein. 
Should we assert 'complement component C1q binding' as having an input of ' complement component C1q complex'?
'complement component C1q binding' is also classified as an 'opsonin binding' and a 'complement binding'.

**Comment by ukemi** [comment]
I geuss it comes down to whether for the binding term we are referring to binding the complex or any C1q protein. Which makes more sense for the binding term? Seems like we actually meant the complex.

**Comment by addiehl** [comment]
I agree we meant the complex.  We might need to make the name and definition clearer.

**Comment by ukemi** [comment]
I can do that. It should be straightforward. @addiehl, with that in mind, does the opsonin parent still make sense? Sorry, not my field.

**Comment by addiehl** [comment]
Yes, the opsonin parent make sense, because the c1q complex bound to an antibody or antibodies (which are themselves bound to some antigen, for instance on a bacterial surface), acts as an opsonin, a trigger for phagocytosis, via a c1q receptor (of which there are several, PMID:10844516) on a macrophage.

**Comment by bmeldal** [comment]
Ah, I look at QuickGO so don't see those relations. yes, the input you suggested makes sense.

We might have to add "complex" to the binding term: complement component C1q COMPLEX binding

I'm creating a google sheet for all the complement complexes:
https://docs.google.com/spreadsheets/d/1n7TW1dmVqK1n25StDrfXlhRwWH8XJLQuq84r8F-5_kc/edit?usp=sharing
(under construction!)
I will use this as my curation guide and add to it as I go along.

**Comment by ukemi** [comment]
OK. I'll take care of the C1q for now. When you wan the others, can you open separate tickets and assign them to me and if he doesn't mind ping @addiehl. I'll modify the terms and defs now and close this ticket.

**Comment by bmeldal** [comment]
Perfect!
Yes, will create new tickets for each request and tag Alex for info.
